ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_program_factory.cpp

```diff
@@ -119,6 +120,7 @@ UnaryProgramFactory::cached_program_t UnaryProgramFactory::create(
                 .math_fidelity = MathFidelity::HiFi4,
                 .fp32_dest_acc_en = args.fp32_dest_acc_en,
                 .unpack_to_dest_mode = unpack_to_dest_mode,
+                .bfp8_pack_precise = true,
```

Comment:
It seems like this bfp_pack_precise could be inferred.

Reply:
I've set it based on the op and dataformat in copy.hpp and unary.cpp